### ISSUE TITLE
Interferes With Other Extensions

### ISSUE THREAD

**Issue by mdlattimore** [Hardeepsingh980/pyscript#7]
Enabling this extension appears to prevent the "Open In Default Browser" (peakchen90) and "Live Server" (Ritwick Dey) extensions from operating normally. Specifically, it removes both from the context menu.

**Comment by THEGOLDENPRO** [comment]
yep, I can confirm that the Live Server does indeed break when this extension is enabled. Quite sad. 😔
